tracer/src/Datadog.Trace/Ci/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -270,12 +283,33 @@ private int WriteMetrics(ref byte[] bytes, int offset, Span span, ITags tags, IT
             offset = tagWriter.Offset;
             count += tagWriter.Count;
 
-            if (span.IsRootSpan && _processIdValueBytes is not null)
+            if (span.IsRootSpan)
             {
-                // add "process_id" tag
-                count++;
-                offset += MessagePackBinary.WriteStringBytes(ref bytes, offset, _processIdNameBytes);
-                offset += MessagePackBinary.WriteRaw(ref bytes, offset, _processIdValueBytes);
+                if (_processIdValueBytes is not null)
+                {
+                    // add "process_id" tag
+                    count++;
+                    offset += MessagePackBinary.WriteStringBytes(ref bytes, offset, _processIdNameBytes);
+                    offset += MessagePackBinary.WriteRaw(ref bytes, offset, _processIdValueBytes);
+                }
+
+                // add "_sampling_priority_v1" tag
+                if (span.Context.TraceContext.SamplingPriority is { } samplingPriority)
```

Comment:
I guess the ciapp serializer doesn't have the concept of chunk orphans etc. Should it? 🤔 Question for @tonyredondo I guess

Reply:
Oh, thanks for reminding me! I meant to ask @tonyredondo about sampling in ciapp.